server/channels/db/migrations/postgres/000122_remove_upload_file_permission.up.sql

```diff
@@ -0,0 +1,31 @@
+DO $$
+<<remove_upload_file_permission>>
+DECLARE
```

Comment:
Glad to see my stored proc being used :). Just wanted to confirm, you have tested this out and it works as expected right?